### ISSUE TITLE
integrate "dat publish"

### ISSUE THREAD

**Issue by yoshuawuyts** [dat-land/dat-desktop#332]
As per our call last week: once the datproject.org api is ready, and we have login (#134)  we should integrate "dat publish" functionality

**Comment by yoshuawuyts** [comment]
@karissa saw you removed the `Status: Blocked` label. Could you perhaps link to the API (source/docs) on how to create accounts and publish to datproject.org? Thanks!

**Comment by martinheidegger** [comment]
Noting that #418 would take care of this.

**Comment by martinheidegger** [comment]
Closing because it looks like a duplicate of #59 - please reopen if I am mistaken.
